packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -394,4 +565,56 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
   );
 };
 
+type MultipleComboboxProps<T> = {
+  selectedItems?: SelectOption<T>[];
+  selectedItemsRenderer?: (options: SelectOption<T>[]) => JSX.Element;
+  tagRenderer: (item: SelectOption<T>) => JSX.Element;
+  mRef: React.ForwardedRef<HTMLDivElement>;
+};
+
+const MultipleSelectedContainer = <T,>({
+  selectedItems,
+  selectedItemsRenderer,
+  tagRenderer,
+  mRef,
```

Comment:
what is "mRef"?

Reply:
Did not found a way to add forwardRef with <T,> type. Found that I can add custom ref props, so added mRef.